_posts/en/newsletters/2020-09-16-newsletter.md

```diff
@@ -0,0 +1,218 @@
+  transaction][russell tx].
+
+- [C-Lightning #3973][] adds support for the accepter side of dual-funded
+  channels.  These are channels where both participants contribute funds
+  rather than just a single initiator contributing funds (see
+  Newsletters [#22][news22 dual] and [#83][news83 dual]).  The initial implementation of the opener side
+  is still being worked on, although there's a [draft][neigut
+  interactive funding] of the related specification changes.
+
+- [C-Lightning #3870][] implements a *scorched earth* fee bumping
+  mechanism for penalty transactions.  If the remote peer broadcasts an
+  old channel state, the local node can use the corresponding revocation
+  key to create a penalty transaction that spends all of the funds
+  claimed by the non-compliant peer.  To avoid this, the peer might
+  attempt to bribe miners to ignore the honest node's penalty
+  transaction.  With a scorched earth policy, the honest node
+  fee bumps its penalty transaction so that it pays its entire value to
+  fees---ensuring the non-compliant peer does not profit from its
+  attempted theft.  In theory, if thieves know they cannot profit from
+  an attack, they won't try it.  This PR seems to have been
+  [inspired][C-Lightning #3832] by a Lightning-Dev mailing list
+  discussion in July, see [Newsletter #104][news104 scorched earth].
+
+    The implementation of the scorched earth policy C-Lightning uses is
+    to periodically fee bump the penalty transaction, eventually
+    replacing it with a transaction that spends all of its value to
+    fees.  Anyone else implementing a similar policy may want to review
+    this code for the way it addresses subtle potential issues such as
+    minimum transaction sizes (see [Newsletter #99][news99 min]) and a
+    year-old change to a maximum fee API in Bitcoin Core  (see [Newsletter
+    #39][news39 maxfeerate]).
+
+- [LND #4310][] adds the ability to create and use profiles with
+  `lncli`.  A profile automatically applies its saved parameters to all
+  connections, making it useful for people who always use the same
+  non-default parameters or who connect to multiple servers.  For
+  example, instead of calling `lncli --rpcserver=10019 --network=regtest
+  getinfo` a user may create a profile and then use just `lncli -p test
+  getinfo`.  The PR additionally allows LND's authentication credentials
+  ("macaroons") to optionally be stored in the profile using encryption.
+
+- [LND #4558][] updates LND's existing [anchor outputs][topic anchor
+  outputs] implementation to match the latest version of the
+  specification, merged three weeks ago (see [Newsletter #112][news112
+  bolts688]).  It also removes anchor outputs from the set of experimental features
+  with the plan to make it enabled by default in the next major release
+  of LND.
+
+- [Rust-Lightning #618][] C/C++ Bindings FIXME:moneyball
+
+- [Libsecp256k1 #558][] implements [schnorr signature][topic schnorr signatures]
```

Comment:
I think this is too much detail, since we've covered schnorr signatures and taproot many times before, and the links to the topic pages give all this detail (and more). I'd suggest editing down to:

"Libsecp256k1 #558 implements schnorr signature verification and single-party signing over the secp256k1 elliptic curve as standardized in BIP340. Schnorr signatures are a key component of the taproot soft fork proposal, which uses aggregated schnorr signatures for “everyone agrees” key-path spends and schnorr signatures in tapscript verification. Bitcoin Core has correspondingly updated their internal libsecp256k1 tree to incorporate this change."

@dongcarl @harding what do you think?

Reply:
I'm fine with it either way.

Reply:
ok, keep it as is!